bootstrap.sh

```diff
@@ -19,39 +14,110 @@ then
+sed -i.template "s|Country-Template|$JURISDICTION_NAME|g" README.md pyproject.toml .github/workflows/workflow.yml .github/PULL_REQUEST_TEMPLATE.md CONTRIBUTING.md
+
+echo -e "${PURPLE}*  ${PURPLE}Remove bootstrap instructions\033[0m"
 sed -i.template -e "3,${last_bootstrapping_line_number}d" README.md  # remove instructions lines
-sed -i.template "s|country-template|$lowercase_country_name|g" README.md
-sed -i.template "s|https://github.com/openfisca/openfisca-country-template|$URL|g" setup.py
-find . -name "*.template" -type f -delete
 
-set +x
+echo -e "${PURPLE}*  ${PURPLE}Prepare \033[0m${BLUE}README.MD\033[0m${PURPLE} and \033[0m${BLUE}CONTRIBUTING.md\033[0m"
+sed -i.template "s|https://example.com/repository|$REPOSITORY_URL|g" README.md CONTRIBUTING.md
 
-git mv openfisca_country_template openfisca_$lowercase_country_name
+echo -e "${PURPLE}*  ${PURPLE}Prepare \033[0m${BLUE}CHANGELOG.md\033[0m"
+sed -i.template -e "1,${last_changelog_number}d" CHANGELOG.md  # remove country-template CHANGELOG leaving example
 
-git rm bootstrap.sh
+echo -e "${PURPLE}*  ${PURPLE}Prepare \033[0m${BLUE}pyproject.toml\033[0m"
+sed -i.template "s|https://github.com/openfisca/country-template|$REPOSITORY_URL|g" pyproject.toml
+sed -i.template "s|:: 5 - Production/Stable|:: 1 - Planning|g" pyproject.toml
+sed -i.template "s|repository_folder|$REPOSITORY_FOLDER|g" README.md
+find . -name "*.template" -type f -delete
+
+echo -e "${PURPLE}*  ${PURPLE}Rename parent directory to: \033[0m${BLUE}openfisca_$SNAKE_CASE_JURISDICTION\033[0m"
+git mv openfisca_country_template openfisca_$SNAKE_CASE_JURISDICTION
+
+echo -e "${PURPLE}*  ${PURPLE}Remove single use \033[0m${BLUE}bootstrap.sh\033[0m${PURPLE} script\033[0m"
+git rm bootstrap.sh > /dev/null 2>&1
 git add .
-git commit --no-gpg-sign --message 'Customise country-template through script' --author='OpenFisca Bot <bot@openfisca.org>'
-git remote add origin $REPOSITORY_URL.git
+git commit --no-gpg-sign --message "$second_commit_message" --author='OpenFisca Bot <bot@openfisca.org>' --quiet
+
+echo -e "${PURPLE}*  ${PURPLE}Second git commit made to 'main' branch: '\033[0m${BLUE}$second_commit_message\033[0m${PURPLE}'\033[0m"
+echo -e ""
+cd ../openfisca-$NO_SPACES_JURISDICTION_LABEL
+
+echo -e "${YELLOW}* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * \033[0m"
+echo -e "${YELLOW}*\033[0m  Bootstrap complete, don't forget to push to remote repository\033[0m"
+echo -e "${YELLOW}*\033[0m "
+echo -e "${YELLOW}*\033[0m  $ ${BLUE}git remote add origin $REPOSITORY_URL.git\033[0m"
+echo -e "${YELLOW}*\033[0m  $ ${BLUE}git push origin main\033[0m"
+echo -e "${YELLOW}*\033[0m "
+echo -e "${YELLOW}*\033[0m  Then refer to the project's \033[0m${BLUE}README.md\033[0m"
+echo -e "${YELLOW}*\033[0m"
+echo -e "${YELLOW}* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * \033[0m"
 
-echo '************'
-echo '* All set! *'
-echo '************'
+exec bash
```

Comment:
This starts a `bash` session without loading `.profile` and other session-specific setup, which is very confusing.

Reply:
Yes, I have not found another solution automatically orientating the user to the newly created directory. I'm open to suggestions on this. We can revert which leaves them at a bash prompt located in a non existent folder which might also be confusing.